### PR TITLE
Update to 3.1.5

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
Rerendering seems to have deleted the appveyor file.

File: appveyor.yml [hunk 1]
```diff
@@ -1,64 +0,0 @@
-
```
Comment (jakirkham) [inline appveyor.yml]: So I don't think this file was suppose to be removed, but I'm guessing this a product of our lack of `vc` support. Maybe try adding `python` to `requirements/build` for the re-rendering and see if that adds this file back. Though make sure not to commit the `python` change.

Comment (isuruf) [reply appveyor.yml]: I guess it doesn't hurt to add `python` to `requirements/build` and commit, right?

Comment (jakirkham) [reply appveyor.yml]: I can't remember why we didn't go that route, but I thought there was a reason.

Comment (jakirkham) [reply appveyor.yml]: In any event, I have been pushing for this to get fixed. So am hopeful that we won't have to go through this in the future.

Comment (jakirkham) [inline appveyor.yml]: That's weird. Actually I don't lose AppVeyor at all when re-rendering and it appears we do have `python` in `build` already. 😕

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Any idea what is going on with this [AppVeyor build](https://ci.appveyor.com/project/conda-forge/mpfr-feedstock/build/1.0.15/job/3h60brw9blu7j0mr#L1529), @msarahan ? It seems unable to activate the test environment.

**Comment by isuruf** [comment]
This seems to be a problem with MPIR. This patch fixes it, but there's been no new MPIR release yet. https://github.com/wbhart/mpir/commit/171de8daed722b06ad4c99e4b6da99480bc13dd3

**Comment by xantares** [comment]
maybe it can be backported, same as we included the long-long fix

**Comment by isuruf** [comment]
@xantares, see conda-forge/mpir-feedstock#13

**Pull request merged**